src/foundry/foundry.js/applications/formApplications/userConfig.d.ts

```diff
@@ -0,0 +1,51 @@
+import { ConfiguredDocumentClass } from '../../../../types/helperTypes';
+import { UserData } from '../../../common/data/data.mjs';
+
+declare global {
+  /**
+   * The User Configuration application provides a form used to allow the current client to
+   * edit preferences and configurations about a User entity (typically their own).
+   * @typeParam P - the type of the options object
+   */
+  class UserConfig<P extends FormApplication.Options = FormApplication.Options> extends FormApplication<
```

Comment:
This actually extends `DocumentSheet`, which is not yet typed correctly. Either do that first, or at least add a TODO about this.